Podcast/EpisodeDetailViewController.swift

```diff
@@ -8,100 +8,84 @@
 
 import UIKit
 
-class EpisodeDetailViewController: ViewController, EpisodeDetailHeaderViewCellDelegate, UITableViewDelegate, UITableViewDataSource {
-    
-    var tableView: UITableView!
+class EpisodeDetailViewController: ViewController, EpisodeDetailHeaderViewDelegate {
+
+    let marginSpacing: CGFloat = EpisodeDetailHeaderView.marginSpacing
     var episode: Episode?
-    var comments: [Comment] = Array.init(repeating: Comment(episodeId: "", creator: "Mark Bryan", text: "Great point here!!", creationDate: "5 months ago", time: "5:13"), count: 10)
-    var episodeDetailViewHeaderHeight: CGFloat = 0
+    var headerView: EpisodeDetailHeaderView = EpisodeDetailHeaderView()
+    var episodeDescriptionView: UITextView = UITextView()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        tableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height), style: .plain)
-        tableView.dataSource = self
-        tableView.delegate = self
-        tableView.register(CommentsTableViewCell.self, forCellReuseIdentifier: "CommentsTableViewCellIdentifier")
-        tableView.register(EpisodeDetailHeaderViewCell.self, forCellReuseIdentifier: "EpisodeDetailHeaderViewCellIdentifier")
-        mainScrollView = tableView
-        view.addSubview(tableView)
-    }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        tableView.reloadData()
-        tableView.setNeedsLayout()
-        tableView.layoutIfNeeded()
-        tableView.reloadData()
-    }
-    
-    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        if indexPath.section == 0, let cell = tableView.dequeueReusableCell(withIdentifier: "EpisodeDetailHeaderViewCellIdentifier") as? EpisodeDetailHeaderViewCell, let episode = episode {
-            cell.setupForEpisode(episode: episode)
-            cell.delegate = self
-            cell.layoutSubviews()
-            episodeDetailViewHeaderHeight = cell.frame.height
-            return cell
-        } else if let cell = tableView.dequeueReusableCell(withIdentifier: "CommentsTableViewCellIdentifier") as? CommentsTableViewCell  {
-            cell.setupForComment(comment: comments[indexPath.row])
-            return cell
+        view.backgroundColor = .offWhite
+        
+        episodeDescriptionView.isEditable = false
+        episodeDescriptionView.font = ._14RegularFont()
+        episodeDescriptionView.textColor = .charcoalGrey
+        episodeDescriptionView.showsVerticalScrollIndicator = false
+        episodeDescriptionView.backgroundColor = .clear
+        episodeDescriptionView.contentInset = UIEdgeInsetsMake(marginSpacing / 2, marginSpacing, marginSpacing, marginSpacing)
+        view.addSubview(episodeDescriptionView)
+        mainScrollView = episodeDescriptionView
```

Comment:
![image](https://user-images.githubusercontent.com/20246620/32679177-6f0c91f6-c633-11e7-8359-6153cbe3493e.png)
the description scrollview starts in the middle of the episode description

Reply:
from the profile, tapping on a recommended episode does this:
![image](https://user-images.githubusercontent.com/20246620/32679977-9f34488a-c636-11e7-8d65-62587ca12308.png)

Reply:
This is weird. Are you still on iPhone 7 small screen? What was the iphone you used to get this bug?

Reply:
it was on the iphone 7 simulator